ai-minter/.env.example

```diff
@@ -1 +1,3 @@
-REPLICATE_API_TOKEN=    # your replicate api token goes here
\ No newline at end of file
+REPLICATE_API_TOKEN=    # your replicate api token goes here
```

Comment:
the ENV KEYS need to be NEXT_PUBLIC_KEY so it can be read on the clientside, now is reading the fallback always check here
https://github.com/Mintbase/templates/blob/0d9b4bb9c3ede576e0bf8b84124c796697f7c02b/minter/src/config/setup.ts